terraform/modules/ecs/ecs.tf

```diff
@@ -34,6 +34,23 @@ resource "aws_ecs_service" "web" {
   depends_on = ["aws_iam_role.ecs_role"]
 }
 
+resource "aws_ecs_service" "logspout" {
+  name            = "logspout-${var.environment}"
+  cluster         = "${aws_ecs_cluster.cluster.id}"
+  task_definition = "${aws_ecs_task_definition.logspout.arn}"
+  desired_count   = "${var.ecs_logspout_task_count}"
+
+  deployment_minimum_healthy_percent = 50
+
+  placement_constraints {
+    type = "distinctInstance"
```

Comment:
I see the commit message says 'Places one container per instance' however does this also ensure that no matter the `desired_count` we only ever have 1 container running, never more? Thinking about the potential to misconfigure this and end up with multiple containers duplicating all logs to Papertrail.

Reply:
If the `desired_count` is higher than the current number of instances running, the service will fail, resulting in no containers being launched.
If the `desired_count` is less than the current number of instances running, the containers will be launched, but there will be instances without the container running.
`desired_count` needs to be stepped up and down along with the same metric that autoscales the instances